force-app/main/default/classes/ADDR_Addresses_UTIL.cls

```diff
@@ -44,6 +44,8 @@ public with sharing class ADDR_Addresses_UTIL {
     /* @description map from Country Code to Country Name */
     private static Map<String, String> validCountriesByCode = new Map<String, String>();
 
+    private final static Set<String> INACTIVE_ADDRESS_FIELDS_PREFIX = new Set<String>{ 'Mailing', 'Billing' };
```

Comment:
Using a constant here is a smart play. Perhaps add a comment here about how we are only supporting inactive address for Mailing Address on Contact and Billing Address on Account or extract logic checks to a method to make it abundantly clear what we are doing and why.